libbroker/broker/internal/peering.cc

```diff
@@ -119,18 +126,20 @@ void peering::on_bye_ack() {
   bye_timeout_.dispose();
 }
 
-void peering::force_disconnect() {
+void peering::force_disconnect(const std::string reason) {
```

Comment:
```suggestion
void peering::force_disconnect(std::string reason) {
```

Adding `const` here turns the `move` below to a copy.